developer_guides/unit_tests.rst

```diff
@@ -84,6 +84,43 @@ Preparing cmocka package
 
       ./configure --with-cmocka-prefix=/home/<you>/cminstall ...
 
+Wrapping objects for unit tests
+******************************
+
+If you need to mock a symbol, define it in a unit test and include the .h file. 
+There are 2 cases where this isn't possible:
+
+*	Static functions in headers(those most probably are inline short functions
+	and don't have to be mocked)
+
+*	Static functions that are in the same file as tested functionality and are
+	exceedingly large so they can't be tested as one functionality. 
+
+Whatever the reason mocking of those symbols can be done by using --wrap linker
+functionality. To wrap the symbol follow these steps:
+
+#. Create mocked symbol named __wrap_symbol_name
+
+#. Pass instruction for the linker -Wl, --wrap=symbol_name during compilation.
+
+Now every symbol call to symbol_name will call __wrap_symbol_name.
+
+Instructions can be passed to the linker in the SOF UT environment using
+CFLAGS, however they should be passed in separate variables in the makefile.
+
+Example:
+
+	.. code-block:: bash

```

Comment:
Line 113: .. code-block:: bash should not be indented just the lines following it.